end_to_end/1-data-prep-e2e.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>Use sagemaker.Session to automatically detect region</p><pre class="ql-syntax" spellcheck="false">import sagemaker

region = sagemaker.Session().boto_region_name
print("AWS Region: {}".format(region))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>we need to explicitly call out the region and have them deliberately set it.</p>